mmdet/datasets/transforms/geometric.py

```diff
@@ -532,6 +530,13 @@ def _transform_seg(self, results: dict, mag: float) -> None:
             border_value=self.seg_ignore_label,
             interpolation='nearest')
 
+    def _transform_bboxes(self, results: dict, mag: float) -> None:
+        """Rotate the bboxes."""
+        img_shape = results['img_shape']
+        center = ((img_shape[1] - 1) * 0.5, (img_shape[0] - 1) * 0.5)
+        results['gt_bboxes'].rotate_(center, mag)
+        results['gt_bboxes'].clip_(img_shape)
+
```

Comment:
Delete. All geometric transform will reuse `_transform_bboxes` of `GeomTransform`.